ui/src/utils/feedback.ts

```diff
@@ -0,0 +1,26 @@
+import {PaperId} from "../state";
+import queryString from "querystring";
+
+export function createFeedbackLink(
+  paperId?: PaperId | undefined,
+  extraContext?: object
+) {
+  /*
+   * The URL and field ids below are generated by Google. The identifiers
+   * are opaque to us, which means they could change (say, if we modify
+   * the assocaited form). If this proves to be an issue we'll have to think
+   * through alternative feedback mechanisms.
```

Comment:
one idea could be making this configurable in the API/UI at deployment time (like the config file used by the API), but that's a later problem and not a now problem.